maths/3n+1.py

```diff
@@ -1,19 +1,19 @@
-def main():
-    def n31(a):# a = initial number
-        c = 0
-        l = [a]
-        while a != 1:
-            if a % 2 == 0:#if even divide it by 2
-                a = a // 2
-            elif a % 2 == 1:#if odd 3n+1
-                a = 3*a +1
-            c += 1#counter
-            l += [a]
+def n31(a):# a = initial number
```

Comment:
Readability:  Please add two spaces between the code and the comment.

Reply:
Sorry, which comment are you referring to ? `# a = initial number` ? Maybe we could do without the comment and choose a better variable name instead of `a`  ;)

Reply:
The PEP8 rule applies to any line in Python that has both code and a comment.

Here is something slick for you to try:
```python
def n31(initial_number):
    """
    >>> n31(43)  # doctest: +ELLIPSIS
    ([43, ...], 29)
    """
    <the rest of the code goes here>
```
And then on the command line do __python3 -m doctest -v maths/3n+1.py__.  This is a doctest and our __.travis.yml__ file guides our automated testing to test your function on each commit.